ci/scripts/goFmt.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -eux
+
+echo "running gofmt..."
+echo ""
+
+if [ "$(gofmt -d . | wc -l)" -gt 0 ]
+  then
+    gofmt -d .
+    echo "gofmt identified one or more issues with your PR"
+    echo ""
+    exit 1
+fi
```

Comment:
`gofmt` will return exit code 0 unless it fails to run we determine success or failure but the number of lines output from this command > 0 equals failure.